git/ssh_config.go

```diff
@@ -102,6 +95,44 @@ func sshParseConfig(c SSHAliasMap, file io.Reader) error {
 	return scanner.Err()
 }
 
+func (p *parser) addAlias(host, token string) {
+	p.mux.Lock()
+	defer p.mux.Unlock()
+	p.aliasMap[host] = token
+}
+
+func parse(files ...string) SSHAliasMap {
+	var wg sync.WaitGroup
+	p := parser{aliasMap: make(SSHAliasMap)}
+
+	for _, file := range files {
+		wg.Add(1)
+		go func(fileName string) {
+			_ = p.read(fileName)
+			wg.Done()
```

Comment:
Ordinarily, this kind of concurrency in Go is great.

In this case, though, parallelism could lead to indeterministic results when multiple files are scanned. We distinctly want the files to be scanned in the order given so that later files (e.g. the user's personal config file) have precedence over earlier files (e.g. system config).

Please either 1) remove the parallelism or 2) ensure that aliases scanned in later files always have precedence over those scanned in earlier files, regardless of the order that the results came back in.

Reply:
I've removed the concurrency. I tried making each config file has its own `SSHAliasMap` and  in the end I could merge them in the right precedence, but for now I believe it okay have it sequentially. :thinking: